### PR TITLE
Adam for Greg

### PR DESCRIPTION
**Pull request by adamish717**
These can probably be ignored and fixed properly, but better than throwing away my thoughts.

### CODE REVIEW

File: src/_includes/components/contact.pug [hunk 1]
```diff
@@ -6,18 +6,14 @@ section.contact#contact
       dl
         dt Call
         dd
-          a(href="tel:{{site.contact.tel}}") {{ site.contact.tel }}
+          a(href="tel:+1.{{site.contact.tel}}") {{ site.contact.tel }}
```
Comment (gregdaynes) [inline src/_includes/components/contact.pug]: Thanks! Except this points out a bigger flaw. Will fix on my end after this is merged in.

File: src/_includes/components/contact.pug [hunk 2]
```diff
@@ -6,18 +6,14 @@ section.contact#contact
...
-      h2 New Office Location
-
-      p During the week of July 24, 2017, KMA moved to a new location that will better suit our growing firm. The new address is:
-
-      p This new location is less than 10 minutes from our current office and very easy to find. We are located next to the Lantzville Municipal Hall on Lantzville Road.
-
-      p Our phone number has changed to (250) 390-4131 but our old number will be automatically forwarded to the new number. Our email contact information will remain the same.
+      h4 Office
+      p 7190 Lantzville Road <br> Lantzville, BC V0R 2H0 <br> Fax: +1.250.390.2832
+      p Mailing Address: <br> P.O. Box 70 <br> Lantzville, BC V0R 2H0
```
Comment (gregdaynes) [inline src/_includes/components/contact.pug]: Can we update these to be microdata
https://schema.org/address

File: src/_includes/components/floating_buttons.pug [hunk 3]
```diff
@@ -2,5 +2,5 @@
   .inner
     a.button.button--accent(href="/services/tax-forms") Tax Forms
 
-  .inner
-    a.button(href="/#email-capture") Sign-Up Now
+  //- .inner
+  //-   a.button(href="/#email-capture") Sign-Up Now
```
Comment (gregdaynes) [inline src/_includes/components/floating_buttons.pug]: Remove commented lines

File: src/_includes/components/header.pug [hunk 4]
```diff
@@ -3,23 +3,23 @@ header.site-header
     a.site-logo(href="/") {{ site.name }}
 
     nav.secondary-nav
-      a.nav--tel(href="#")
+      a.nav--tel(href="tel:+1.{{ site.contact.tel }}")
```
Comment (gregdaynes) [inline src/_includes/components/header.pug]: Will fix in another pr

File: src/_includes/components/header.pug [hunk 5]
```diff
@@ -3,23 +3,23 @@ header.site-header
...
         i.icon--upload
         span Secure File
         | Upload
 
-      a.nav--facebook(href="#")
-        i.icon--facebook
-        span.image-replacement Facebook
+      //- a.nav--facebook(href="#")
+      //-   i.icon--facebook
+      //-   span.image-replacement Facebook
```
Comment (gregdaynes) [inline src/_includes/components/header.pug]: remove

File: src/_includes/components/header.pug [hunk 6]
```diff
@@ -3,23 +3,23 @@ header.site-header
...
+      a.nav--twitter(href="https://twitter.com/kmacpaca")
         i.icon--twitter
         span.image-replacement Twitter
 
-      a.nav--linkedin(href="#")
-        i.icon--linkedin
-        span.image-replacement Linkedin
+      //- a.nav--linkedin(href="#")
+      //-   i.icon--linkedin
+      //-   span.image-replacement Linkedin
```
Comment (gregdaynes) [inline src/_includes/components/header.pug]: remove

File: .gitignore [hunk 7]
```diff
@@ -2,3 +2,5 @@ node_modules
 tmp
 .DS_Store
 .sass-cache/
+.pug-cache/
+output/
```
Comment (gregdaynes) [inline .gitignore]: Do we not want to keep the output - at least for a faux Artifact?

File: src/_includes/components/contact.pug (hunk 2 above)
Comment (gregdaynes) [reply src/_includes/components/contact.pug]: Second thought - I'll take care of this

**Comment by adamish717** [comment]
Worst PR I've ever seen. CANCELLED!

**Pull request closed without merging**